### PR TITLE
feat(test): Byzantine test

### PR DESCRIPTION
**Pull request by rev-chaos**
**What this PR does / why we need it**:
RT

**Which issue(s) this PR fixes**:

Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rev-chaos** [comment]
Have found 3 bugs so far:
1. multi-sig service rlp decode signature panic!  Kao fixed in https://github.com/nervosnetwork/muta/pull/411.
2. overlord rlp decode proposal panic! The cause is still being investigated.
3. overlord straightly inserts proposal which height equals current height but round is not current round without check, this will cause liveness problem in byzantine attack.

**Comment by yejiayu** [comment]
Nice! add the byzantine test to ci?

File: byzantine/generators.toml [hunk 1]
```diff
@@ -0,0 +1,448 @@
+interval = 500
```
Comment (yejiayu) [inline byzantine/generators.toml:1]: Suggest splitting config for diff cases

File: byzantine/generators.toml [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+probability = 1.0
+num_range = [1, 10]
+priority = "High"
+
+[[list]]
+req_end = "/rpc_call/mempool/pull_txs"
+msg_type = { PullTxs = "InvalidSender" }        # not cause bugs
+probability = 1.0
+num_range = [1, 10]
+priority = "High"
```
Comment (yejiayu) [inline byzantine/generators.toml:448]: new line

**Review by KaoImin — APPROVED** [review]

**Pull request merged**